src/queries.js

```diff
@@ -15,73 +15,73 @@ function windowQuery(windowbucket, afkbucket, appcount, titlecount, filterAFK) {
-  var browser_appnames = "";
-  if (browserbucket.endsWith("-chrome")){
-    browser_appnames = '["Google-chrome", "chrome.exe", "Chromium", "Google Chrome"]';
-  } else if (browserbucket.endsWith("-firefox")){
-    browser_appnames = '["Firefox", "Firefox.exe", "firefox", "firefox.exe"]';
+  let browserAppnames = '';
+  if (browserbucket.endsWith('-chrome')) {
+    browserAppnames = '["Google-chrome", "chrome.exe", "Chromium", "Google Chrome"]';
+  } else if (browserbucket.endsWith('-firefox')) {
+    browserAppnames = '["Firefox", "Firefox.exe", "firefox", "firefox.exe"]';
   }
 
   return [
-    'events = flood(query_bucket("' + browserbucket + '"));',
-    'window_browser = flood(query_bucket("' + windowbucket + '"));',
-    'window_browser = filter_keyvals(window_browser, "app", ' + browser_appnames + ');',
+    `events = flood(query_bucket("${browserbucket}"));`,
+    `window_browser = flood(query_bucket("${windowbucket}"));`,
+    `window_browser = filter_keyvals(window_browser, "app", ${browserAppnames});`,
   ].concat(filterAFK ? [
-    'not_afk = flood(query_bucket("' + afkbucket + '"));',
+    `not_afk = flood(query_bucket("${afkbucket}"));`,
     'not_afk = filter_keyvals(not_afk, "status", ["not-afk"]);',
     'window_browser = filter_period_intersect(window_browser, not_afk);',
   ] : [])
-  .concat([
-    'events = filter_period_intersect(events, window_browser);',
-    'events = split_url_events(events);',
-    'urls = merge_events_by_keys(events, ["domain", "url"]);',
-    'urls = sort_by_duration(urls);',
-    'urls = limit_events(urls, ' + count + ');',
-    'domains = split_url_events(events);',
-    'domains = merge_events_by_keys(domains, ["domain"]);',
-    'domains = sort_by_duration(domains);',
-    'domains = limit_events(domains, ' + count + ');',
-    'chunks = chunk_events_by_key(events, "domain");',
-    'duration = sum_durations(events);',
-    'RETURN = {"domains": domains, "urls": urls, "chunks": chunks, "duration": duration};',
-  ]);
+    .concat([
+      'events = filter_period_intersect(events, window_browser);',
+      'events = split_url_events(events);',
+      'urls = merge_events_by_keys(events, ["domain", "url"]);',
+      'urls = sort_by_duration(urls);',
+      `urls = limit_events(urls, ${count});`,
+      'domains = split_url_events(events);',
+      'domains = merge_events_by_keys(domains, ["domain"]);',
+      'domains = sort_by_duration(domains);',
+      `domains = limit_events(domains, ${count});`,
+      'chunks = chunk_events_by_key(events, "domain");',
+      'duration = sum_durations(events);',
+      'RETURN = {"domains": domains, "urls": urls, "chunks": chunks, "duration": duration};',
+    ]);
 }
 
-function editorActivityQuery (editorbucket, limit){
+function editorActivityQuery(editorbucket, limit) {
   return [
-    'editorbucket = "' + editorbucket + '";',
+    `editorbucket = "${editorbucket}";`,
```

Comment:
Personal opinion is that I hate backticks since they are hard to write on an swedish ISO keyboard lol, but it's easier to read so it's worth it.

Reply:
Oh, that is unfortunate :disappointed: